File: app/src/main/java/com/mutenlab/sudoit/util/PermissionHelper.java
package com.mutenlab.sudoit.util;

import android.content.Context;
import android.content.pm.PackageManager;
import android.support.v4.content.ContextCompat;

/**
 * @author icerrate
 */
public class PermissionHelper {

    public static boolean checkPermissions(Context context, String[] permissions) {
        for (String permission : permissions) {
            if (ContextCompat.checkSelfPermission(context, permission) != PackageManager.PERMISSION_GRANTED) {
                return false;
            }
        }
        return true;
    }
}

File: app/src/main/kotlin/com.mutenlab.sudoit/utils/PermissionHelper.kt
package com.mutenlab.sudoit.utils

import android.content.Context
import android.content.pm.PackageManager
import android.support.v4.content.ContextCompat

/**
 * @author Ivan Cerrate.
 */

object PermissionHelper {

    fun checkPermissions(context: Context, permissions: Array<String>): Boolean {
        for (permission in permissions) {
            if (ContextCompat.checkSelfPermission(context, permission) != PackageManager.PERMISSION_GRANTED) {
                return false
            }
        }
        return true
    }

}
